ci/deploy-image.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -Eeuxo pipefail
+
+function deploy() {
+  image=${1}
+  mvn -q -B -V com.google.cloud.tools:jib-maven-plugin:1.3.0:build \
+    -Dimage="${image}" \
+    -Djib.to.auth.username="${DOCKER_USER}" \
+    -Djib.to.auth.username="${DOCKER_PASS}"
+}
+
+function main() {
+  mvn -q -B -V package -DskipTests=true
```

Comment:
if we use "deploy", let's tell travis to keep the filesystem from the build

Reply:
noticed it independently :)